### PR TITLE
correct some words

### PR DESCRIPTION
**Pull request by liliVicky**
Signed-off-by: liwenqi <vikilwq@zju.edu.cn>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Review by AkihiroSuda — COMMENTED** [review]
LGTM, but please consider squashing with https://github.com/docker/docker/pull/29575 ?

File: docs/reference/commandline/build.md [hunk 1]
```diff
@@ -262,7 +262,7 @@ Successfully built 377c409b35e4
 This sends the URL `http://server/ctx.tar.gz` to the Docker daemon, which
 downloads and extracts the referenced tarball. The `-f ctx/Dockerfile`
 parameter specifies a path inside `ctx.tar.gz` to the `Dockerfile` that is used
-to build the image. Any `ADD` commands in that `Dockerfile` that refer to local
+to build the image. Any `ADD` commands in that `Dockerfile` that refers to local
```
Comment (thaJeztah) [inline docs/reference/commandline/build.md]: I don't think this is correct; "any commands that _refer_ to .. " (commands is plural)

File: project/ISSUE-TRIAGE.md [hunk 2]
```diff
@@ -129,4 +129,4 @@ following labels to indicate their degree of priority (from more urgent to less
 | priority/P2 | Normal priority: default priority applied.                                                                                        |
 | priority/P3 | Best effort: those are nice to have / minor issues.                                                                               |
 
-And that's it. That should be all the information required for a new or existing contributor to come in an resolve an issue.
+And that's it. That should be all the information required for a new or existing contributor to come in a resolve an issue.
```
Comment (thaJeztah) [inline project/ISSUE-TRIAGE.md]: this should be "and"

File: docs/reference/builder.md [hunk 3]
```diff
@@ -1754,6 +1754,6 @@ FROM ubuntu
 RUN echo moo > oink
 # Will output something like ===> 695d7793cbe4
 
-# You᾿ll now have two images, 907ad6c2736f with /bar, and 695d7793cbe4 with
+# You'll now have two images, 907ad6c2736f with /bar, and 695d7793cbe4 with
```
Comment (thaJeztah) [inline docs/reference/builder.md]: Curly quote is not incorrect here, but for CLI examples / comment, I'm ok to change it (http://practicaltypography.com/straight-and-curly-quotes.html)

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: docs/reference/builder.md (hunk 3 above)
Comment (thaJeztah) [inline docs/reference/builder.md]: FWIW; curly quotes are not incorrect here, but given that it's a comment I'm ok with changing it here for readability (see http://practicaltypography.com/straight-and-curly-quotes.html)

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**